ccm_web/server/src/canvas/canvas.model.ts

```diff
@@ -0,0 +1,39 @@
+import { BelongsTo, Column, ForeignKey, Model, Table } from 'sequelize-typescript'
+import { DataTypes, Optional } from 'sequelize'
+
+import { User } from '../user/user.model'
+
+interface CanvasTokenAttributes {
+  id?: bigint
+  userId: bigint
+  accessToken: string
+  refreshToken: string
+}
+
+interface CanvasTokenCreationAttributes extends Optional<CanvasTokenAttributes, 'id'> {}
+
+@Table({
+  tableName: 'canvasToken',
```

Comment:
just a thought, how about pulling these table name from a separate file since this is hand coded in 2 different place if not carefully can run into issue while development

Reply:
@pushyamig, I like the idea of not repeating things, but I'm not sure the payoff or benefit is there in this case. This string is used in two places (the migration file and the model file). In the case of the migration file, since that's a static thing representing a change and how to apply and un-apply it, I don't think it makes sense to import a variable that might be changed by editing another file. Does that make sense?

Reply:
ok